README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 1
+# Short Bio on Tarryn Rennie 
+
+### I decided that for this assignment I was going to tell you **15 things** that you wouldn't know just from meeting me, instead of just the usual tired-out blurb. 
+
+1. I am a Zimbabwean, born and raised. 
+1. I almost qualified the 2016 Olympics for my national team, falling just short. 
+1. I swam in college but retired after my collegiate career was done. I am now a "swammer."
+1. I have a cat, Addie, who I found in a dumpster last January and decided I couldn't part with (much to my former roommates' consternation). I had wanted a dog, so decided to try train her... Needless to say, it didn't go well BUT she does come when I whistle, which is pretty cool. 
+1. I can draw and paint. I really like doing portraiture. 
+1. I have one sibling, a brother named Piers. We are always mistaken for each other. I am often called sir, and he is called "ma'am," and yes, we don't know why. 
+1. My favourite book is called *"The Power of One."* I highly recommend it. 
+1. I don't really like movies. They're too long and I get bored half way through. Saying that, my guilty pleasure are shows like *"The Good Wife,"* or *"Homeland."*
+1. I am currently trying to learn French. 
+1. If I could change one thing about myself, I would give myself musical talent. 
+1. I love to cook, though I always take liberties with a recipe. Sometimes it works. Sometimes... not so much. 
+1. My weakness is soft gummy candy. Or marshmellows. 
+1. When I was growing up, I wanted to be a fighter pilot. This was until I realised I have far from perfect vision, and I get motion sick. 
+1. I absolutely *love* stationery. It's a problem and I am convinced I keep the Hyde Park Target afloat on pen sales alone. 
+1. Finally, nothing makes me happier than [sunflowers](https://fineartamerica.com/featured/3-sunflower-summer-sunset-landscape-with-blue-skies-matthew-gibson.html). They are just the prettiest things in the world. 
```

Comment:
Nice work on embedding a link! I was able to click on it successfully (also beautiful photo).